com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md

```diff
@@ -40,6 +41,20 @@ To display the current active item independently of the debug window:
 - **Xbox controller**: Press the X button.
 - **PlayStation controller**: Press the Square button.
 
+<a name="RenderGraphPanel"></a>
+
+## Render Graph panel
+
+The **Render Graph** panel has tools that you can use to debug the [Render Graph](https://docs.unity3d.com/2020.2/Documentation/Manual/render-graph.html) used by HDRP.
+
+| **Debug Option**  | **Description**                                              |
+| ----------------- | ------------------------------------------------------------ |
+| **Clear Render Targets at creation**  | Enable the checkbox to clear render targets the first time they are used. |
+| **Disable Pass Culling**              | Enable the checkbox to render passes which have no impact on the final render. |
+| **Immediate Mode**                    | Enable the checkbox to have Render Graph evaluate passes immediately after they are created. |
```

Comment:
Enable the checkbox to make the render graph system evaluate render passes immediately after it creates them.

Reply:
What happens when the checkbox is disabled? Is this when they are evaluated at the end of all the render pass declaration when the graph compiles and executes?

Reply:
Disabled is default behavior. After creation, the unused passes are culled and the remaining one are executed